tests/data/ca/intermediate_server/certs/infuxdb-v1-tls-chain.cert.pem

```diff
@@ -1,98 +0,0 @@
------BEGIN CERTIFICATE-----
```

Comment:
I tried to figure out why the `infuxdb` cert survived and couldn't. If it isn't necessary, I propose deleting it and its associated items. -- It's possible there's a test somewhere (outside this repository?) relating to it...

Reply:
Definitely think it could go, I'll try and confirm.